#! /bin/sh /usr/share/dpatch/dpatch-run
## 33_eximon.binary.dpatch by Andreas Piesk
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: eximon.bin is installed in /usr/lib/exim4/ and not in path.

--- a/OS/eximon.conf-Default
+++ b/OS/eximon.conf-Default
@@ -5,7 +5,7 @@
 # The name of the eximon binary, usually the same as the eximon script,
 # with .bin stuck on the end.
 
-EXIMON_BINARY="${EXIMON_BINARY-$0.bin}"
+EXIMON_BINARY="/usr/lib/exim4/${EXIMON_BINARY-${0##*/}.bin}"
 
 # The remaining parameters are values likely to be changed to suit the
 # user's taste. They are documented in the EDITME file.
